frontend/src/TableLayout.tsx

```diff
@@ -84,22 +86,24 @@ export default function TableLayout({
             >
                 {loading && <LoadingOverlay />}
 
-                <Table
-                    aria-label={label}
-                    noWrap
-                    size="sm"
-                    variant="outlined"
-                    stickyHeader
-                    sx={{
-                        tableLayout: "auto",
-                        border: "none",
-                        "& tr > *": { textAlign: "center" },
-                        ...tableStyle,
-                    }}
-                >
-                    <thead>{header}</thead>
-                    <tbody>{body}</tbody>
-                </Table>
+                {table || (
+                    <Table
```

Comment:
The changes in this file are to let us temporarily live in the weird middle place I've created of using a custom table component for one page, and the MUI table component for the other pages. Sorry, it's not ideal! I'll undo this once I've switched the other pages over.

Reply:
Totally fine! I'm on board, this isn't even that bad. I know we have tracking issues, but maybe worth leaving a TODO or a FIXME as a breadcrumb?

Reply:
Good call since it's an implementation plan that solves issues rather than an explicit issue itself, so it's not being tracked anywhere! Added a reminder TODO